Source/Core/DolphinWX/TASInputDlg.cpp

```diff
@@ -764,11 +786,37 @@ void TASInputDlg::GetValues(u8* data, WiimoteEmu::ReportFeatures rptf, int ext,
 	}
 }
 
+void TASInputDlg::GetValuesCallback(wxCommandEvent& event)
+{
+	if (event.GetEventType() == WIIMOTE_UPDATE_CALLBACK)
```

Comment:
`event.GetEventType()` can be stored to its own variable to reduce redundancy, other than that, it looks fine to me.